mathics/builtin/arithfns/basic.py

```diff
@@ -520,6 +528,8 @@ class Power(BinaryOperator, _MPMathFunction):
     rules = {
         "Power[]": "1",
         "Power[x_]": "x",
+        "Power[I,-1]": "-I",
+        "Power[-1, 1/2]": "I",
```

Comment:
As before, in master, this is not needed. Why is it needed here?

Reply:
Here the previous rule seems to be repeated. I have to check.